### ISSUE TITLE
Bundle JSON schema files in java package

### ISSUE THREAD

**Issue by hrishin** [garethr/kubernetes-json-schema#1]
Referring to discussion on https://github.com/fabric8io/fabric8-maven-plugin/issues/1034 is it possible to create java package (preferable maven  jar would be great) to bundle essential (local) JSON schema files?

**Comment by garethr** [comment]
This sounds like a great idea. I've had some thoughts about better packaging for this content and a Jar sounds like a useful integration point for some folks. If you have ideas about how that would look from the user perspective I'll certainly take a look, or I'd be happy to see a PR.

I've been focusing on a few issues on kubeval recently but I've nearly fixed those I hope. Then I want to come back and refactor/rebuild the tools that generate this repository, including much better packaging and some automation. I've a bunch of ideas so thanks for opening the issue.

**Comment by hrishin** [comment]
yeah here is my initial thought since we are trying to wrap these files in the jar hence program in jar will read the deferenced (standalone) schema, returns a handle to it. Soon I will post a a project for it.
By the way only problem I'm facing is, looks like master of kubernetes OpenAPI  spec has lot of circular dependencies and its failing `openapi2jsonschema` schema util
